_posts/blog/2021-03-29-frost-audit.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: post
+title: "The Zcash Foundation has Finished an Audit of FROST"
+excerpt: "FROST Implementation Audit"
+categories: blog
+tags: [frost]
+Date: 2021-03-29
+author: zfteam
+---
+
+The Zcash Foundation is pleased to announce our completion of an audit for our first implementation [FROST implementation](https://github.com/ZcashFoundation/redjubjub/blob/main/src/frost.rs). This implementation is a first step and does not yet include support for re-randomziation nor the DKG, but these will be features that we will add next (and will complete separate audits for each).
+
+For this audit, we worked with JP Aumasson and Adrian Hamelink from the [Taurus Group](https://www.taurushq.com) and Omer Shlomovits from [ZenGo](https://www.zengo.com/), who have deep experience [evaluating the security](https://eprint.iacr.org/2020/1052) of multi-party signature schemes. We have finished implementing their recommended security measures, and are in the process of fixing
```

Comment:
Please change to "from [Taurus](https://www.taurushq.com)", we just renamed the company to get rid of the "Group"  :)

Reply:
Sure thing.